components/aws/sources/new-scheduled-tasks/README.md

```diff
@@ -24,18 +24,19 @@ You can also listen for these events in your own app / infra, by [subscribing to
 
 <!--ts-->
 
-- [Quickstart](#quickstart)
-- [Prerequisites](#prerequisites)
-  - [An AWS account](#an-aws-account)
-  - [An AWS IAM Policy for creating the necessary AWS resources](#an-aws-iam-policy-for-creating-the-necessary-aws-resources)
-  - [An AWS IAM user, with an access and secret key](#an-aws-iam-user-with-an-access-and-secret-key)
-- [AWS Resources](#aws-resources)
-  - [Estimated AWS Costs](#estimated-aws-costs)
-- [HTTP API](#http-api)
-  - [Scheduling a task](#scheduling-a-task)
-  - [Cancel scheduled tasks](#cancel-scheduled-tasks)
-- [Processing scheduled tasks](#processing-scheduled-tasks)
-- [Example: Schedule a task 30 seconds in the future](#example-schedule-a-task-30-seconds-in-the-future)
+- [AWS Task Scheduler](#aws-task-scheduler)
```

Comment:
@psavkar the ToC generator always includes the `h1` tag but I remove it because it's not a part of the core content. Mind removing and indenting the other sections to the left?

Reply:
done